mfr/settings.py

```diff
@@ -170,3 +170,4 @@ def child(key):
 
 
 SENTRY_DSN = config.get_nullable('SENTRY_DSN', None)
+GOOGLE_ANALYTICS_TRACKING_ID = config.get_nullable('GOOGLE_ANALYTICS_TRACKING_ID', None)
```

Comment:
Can GA script handle `None` tracking ID? I guess so but just want to double check with you.

Reply:
I am not sure. It'll probably throw an error.

Reply:
@cslzchen Confirmed that it would not cause an error when tracking ID is `None`.